pkg/driver/controller_test.go

```diff
@@ -1521,45 +1521,6 @@ func TestCreateVolume(t *testing.T) {
 				checkExpectedErrorCode(t, err, codes.Aborted)
 			},
 		},
-		{
-			name: "fail with missing iopsPerGB parameter",
```

Comment:
Can we add a test to check for success when using `io1/io2` volume with no `iopsPerGb` parameter?

Reply:
I added 2 tests (one for io1, one for io2) to check for success when iops parameter is used instead of iopspergb.